tests/integration/optional_1.phpt

```diff
@@ -4,15 +4,8 @@ require 'vendor/autoload.php';
 
 use Respect\Validation\Validator as v;
 
-var_dump(
-    v::alpha()->validate(''),
-    v::alpha()->validate(null),
-    v::optional(v::alpha())->validate(''),
-    v::optional(v::alpha())->validate(null)
-);
+v::optional(v::alpha())->validate('');
+v::optional(v::alpha())->validate(null);
+
 ?>
 --EXPECTF--
-bool(false)
-bool(false)
-bool(true)
-bool(true)
```

Comment:
The current tests is valid, in such, you can use this new test as another one, like "optional_2.phpt".

You can just update this pull request, reverting this file to the last version.